views/peeps.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <form action= '/peeps' method ='post'>
+    <input type="text" name="peep_filter" placeholder="Write your peep here!">
+    <input type="submit" name="Submit">
+  </form>
+</div>
+
+<ul style="list-style-type:none">
+  <% @peeps.each do |peep| %>
+  ---------
+    <li><%= peep.peep_text  %> Posted on: <%= peep.peep_date  %></li>
+  <% end %>
+</ul>
```

Comment:
Have you thought about the aesthetics of what it looks like at all? Definitely the last thing to consider though